androidsdk/src/test/java/com/wootric/androidsdk/SurveyValidatorTest.java

```diff
@@ -63,6 +63,9 @@ public void checksEligibility_whenNotRecentlySurveyed() throws Exception {
     public void doesNotCheckEligibility_whenRecentlySurveyed() throws Exception {
         User user = testUser();
         EndUser endUser = testEndUser();
+        long createdAtJustNow = new Date().getTime();
```

Comment:
I have to analyse this one more thoroughly since I don't see any conversions from milliseconds to seconds in the whole SDK. Ether they were comparing all dates in milliseconds and the converting to seconds before saving to the server, or this isn't being done at all...